service/history/mutableStateBuilder.go

```diff
@@ -4258,7 +4282,7 @@ func (e *mutableStateBuilder) closeTransactionHandleDecisionFailover(
 	}
 
 	// Handling mutable state turn from standby to active, while having a decision on the fly
-	di, ok := e.GetInFlightDecisionTask()
+	di, ok := e.GetInFlightDecision()
```

Comment:
nit: di=> decisionInfo

Reply:
sure

Reply:
changing di -> decision or otherwise will collide with the decisionInfo type